trucks.js

```diff
@@ -248,9 +252,22 @@ var foodTrucks = [
   }
 ];
 
-/* 
- * return an object from the module with a single method on it: filterByDay
- * that method should take a single parameter that represents the day to filter on
- * use the built-in filter() method to return all trucks that have the day in their
- * schedule
- */
\ No newline at end of file
+function filterByDay(day){
+  var a = foodTrucks.filter(function(truck){
```

Comment:
the `.filter` method returns an array, so you don't need to go through the extra steps to put the result into another array. You can return a directly, or even return the result of calling `filter` directly.